src/components/common/BusySpinner.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, CircularProgress, useTheme } from '@mui/material';
+
+type BusySpinnerProps = {
+  withSkrim?: boolean;
+};
+
+export default function BusySpinner({ withSkrim }: BusySpinnerProps): JSX.Element {
```

Comment:
We have a ProgressCircle component on web-components, but I don't think it's the same as this one

Reply:
This is a bit different, i moved it out into a separate component because we are using in multiple places now. It shows a spinner on top of content but with a blurred skrim. We use it in withdrawals save as well.